src/axom/mir/clipping/ClipField.hpp

```diff
@@ -2248,8 +2133,7 @@ class ClipField
           const int sizeIndex = fragmentData.m_fragmentOffsetsView[index];
           const int nFragments = fragmentData.m_fragmentsView[index];
           const auto zoneIndex = selectedZonesView[index];
-          for(int i = 0; i < nFragments; i++)
```

Comment:
I suppose the original code needs braces for the 1-line for loop. The format making the for loop be single-line is a negative for me.

Reply:
Maybe [AllowShortLoopsOnASingleLine](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#AllowShortLoopsOnASingleLine)=False

Reply:
When we add the braces, it'll become either:

```cpp
for(int i = 0; i < nFragments; i++) { valuesView[sizeIndex + i] = zoneIndex; }
```
or
```cpp
for(int i = 0; i < nFragments; i++) 
{ 
  valuesView[sizeIndex + i] = zoneIndex; 
}
```

Reply:
I much prefer the 2nd version.

Reply:
I agree.